usecases-integration/src/main/java/eu/europeana/cloud/integration/usecases/DatasetHelper.java

```diff
@@ -107,16 +108,21 @@ public void addRecordsToDataset(int numberOfRecords, String datasetName, String
             if (specificCloudId == null) {
                 newCloudId = prepareCloudId(providerId);
             }
-            String version = getVersionFromFileUri(representationName, providerId);
-            for (String tagName : tagNames) {
-                revisionServiceClient.addRevision(newCloudId, representationName, version, revisionName, providerId, tagName);
-            }
+            String uri = addFileToNewRepresentation(representationName, providerId, FILE_CONTENT);
```

Comment:
uri variable is not necessary here, you can pass addFileToNewRepresentation call to getVersionFromFileUri directly